modules/ting_infomedia/ting_infomedia.install

```diff
@@ -65,13 +46,8 @@ function ting_infomedia_update_7000() {
 }
 
 /**
- * Unlock ding_entity_field to allow deletion of field
+ * Unlock fields, see ding_entity_unlock_fields().
  */
 function ting_infomedia_update_7001() {
-  db_update('field_config')
```

Comment:
Update an update function. But if the sites have execute this update. They won't get this updated update?